field_audit/models.py

```diff
@@ -581,6 +581,12 @@ def delete(self, *, audit_action=AuditAction.RAISE):
 
     @validate_audit_action
     def update(self, *, audit_action=AuditAction.RAISE, **kw):
+        """
+        In order to determine what the old and new values of instances within
+        the queryset, one additional fetch on this queryset is performed,
+        resulting in two fetches of items in the queryset and one bulk creation
+        of audit events
```

Comment:
Minor update. I found the initial mention of '2 fetches' confusing since 1 is a fetch and 1 is an update.

```suggestion
        In order to determine what the old and new values of instances within
        the queryset, one additional fetch on this queryset is performed,
        and one bulk creation of audit events.
```

Reply:
Ah interesting. My understanding is there are 2 fetches in addition to the update query, not just 1 ([first one](https://github.com/dimagi/django-field-audit/blob/2036956160bed4bb304fbd8b953b36313f9d3c60/field_audit/models.py#L594) and [second one](https://github.com/dimagi/django-field-audit/blob/2036956160bed4bb304fbd8b953b36313f9d3c60/field_audit/models.py#L604)). This is why I questioned this existing behavior in the description of this PR, because two fetches is definitely unnecessary.

Reply:
oh interesting, I missed that 2nd one. That's very unfortunate that we have to do that. Is it not possible to refactor the `make_audit_event` so that it doesn't rely on having the full instance?

Reply:
Definitely possible and was missed in the initial implementation. I'll take a pass at refactoring it 👍🏻

Reply:
Yeah, I missed this second fetch during my review as well. I agree just the `self.values(*values_to_fetch)` would be the ideal (only) fetch, which would require a bit of a refactor.